framework/auth/cas.py

```diff
@@ -173,6 +168,7 @@ def revoke_tokens(self, payload):
         else:
             self._handle_error(resp)
 
+
```

Comment:
single line break for class methods (https://www.python.org/dev/peps/pep-0008/#blank-lines)